config/application.rb

```diff
@@ -36,5 +36,8 @@ class Application < Rails::Application
 
     # Set URL options to be able to use url_for helpers
     config.action_mailer.default_url_options = { host: ENV.fetch("HOST") }
+
+    # Allowed hosts:
+    config.hosts << ["localhost", ENV.fetch("HOST")]
```

Comment:
Why do we need `localhost` for all environments? It is included in development environment by default.